pages/use-solid.md

```diff
@@ -38,6 +38,7 @@ You can pick a Provider in the following list. However, keep in mind that Solid
 |-----------------------------------|:---------------------------------------------------:|:-------------------------------------------------:|:-------------------:|:--------------------:|
 | [inrupt.net](https://inrupt.net) | [Inrupt, Inc.](https://inrupt.com/terms-of-service) | [Amazon](https://aws.amazon.com) | USA |
 | [solidcommunity.net](https://solidcommunity.net/) | [Solid Project](https://github.com/solid/solidcommunity.net_operations) | [Digital Ocean](https://www.digitalocean.com) | UK |
+| [solidweb.org](https://solidweb.org) | [Solid Grassroots](https://gitlab.com/groups/solidweb.org) | [Hosteurope](https://www.hosteurope.de) | Germany |
```

Comment:
The name "Solid Grassroots" does not seem to be mentioned anywhere on the linked page?

Reply:
yes, @Vinnl it is meant as a description or categorization. we are 3 people momentary with an optional fourth joining us. what' d you think if I link to the members page https://gitlab.com/groups/solidweb.org/-/group_members instead of the project page ?

Reply:
Yeah that sounds good. Alternatively, if you don't generally refer to yourself as a group by the name of "Solid Grassroots", you could just list your names here? Or just something like "solidweb.org administrators" linked to that group overview page.

Reply:
changed to members link now in the patch-2 branch of ewingson. In fact, I like the Grassroots label a lot, cause it defines, how we act and look at ourselves. but as second option - if objections persists - we would be comfort with 'Solid Administrators' as well. what do you think ?

Reply:
No that's fine, I'd just update the GitLab group page to mention that name as well, otherwise it kind of looks like a broken link here.

Reply:
@Vinnl I' ve changed the group description and mentioned 'Grassroots' there

Reply:
Super, thanks!